docs/pages/identity-governance/aws-iam-identity-center/guide.mdx

```diff
@@ -113,6 +115,21 @@ the same [IdP](../../zero-trust-access/sso/sso.mdx) as your current Identity Cen
 external identity source.
 </Admonition>
 
+<Admonition type="info" title="SAML IdP RBAC">
+In this step, Teleport creates a `saml_idp_service_provider` resource for the 
+AWS IAM Identity Center. 
+
+Starting with Teleport version 18 and role version 8, the SAML IdP service provider 
```

Comment:
Nit: Link "role version 8" to the appropriate reference section (we had a table highlighting version differences somewhere)?